### PR TITLE
Events

### LINKED ISSUE #5: Create Events page
Create events page as specified in the design: **NOTE:** We need to decide where the events will be pulled from and how they'll be edited/accessed.

### PR DESCRIPTION
**Pull request by dellisd**
Add an event page.

Still needs actual event data!

### CODE REVIEW

File: src/pages/events.tsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+`
+
+const EventTitle = styled.h1`
+  margin: 5px 0 0 0;
+`
+
+const EventTime = styled.h3`
+  color: var(--light-purple);
+  font-size: 18px;
+
```
Comment (FemmeQueenInSTEM) [inline src/pages/events.tsx:181]: Looks nicer.
```suggestion
font-weight: normal;
```

File: src/components/logoHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  height: 100%;
+
+  justify-content: center;
+
+  color: var(--white);
+  width: var(--mobile-width);
+  @media only screen and (min-width: 1200px) {
+    width: var(--desktop-width);
+    flex-direction: row;
+    font-size: 16px;
```
Comment (FemmeQueenInSTEM) [inline src/components/logoHeader.tsx]: Use `rem` or `em` here.

File: src/components/logoHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    justify-content: left;
+    align-content: center;
+  }
+
+  font-size: 10px;
```
Comment (FemmeQueenInSTEM) [inline src/components/logoHeader.tsx]: And here

File: src/pages/events.tsx [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+        margin-top: 100px;
+    }
+
+    flex-direction: row;
+  }
+`
+
+const EventsHeader = styled.h1`
+  color: var(--dark-purple);
+  font-size: 48px;
```
Comment (FemmeQueenInSTEM) [inline src/pages/events.tsx]: Use `rem`/`em`

File: src/pages/events.tsx [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+  margin: 15px;
+`
+
+const EventTitle = styled.h1`
+  margin: 5px 0 0 0;
+`
+
+const EventTime = styled.h3`
+  color: var(--light-purple);
+  font-size: 18px;
```
Comment (FemmeQueenInSTEM) [inline src/pages/events.tsx]: And here

File: src/pages/events.tsx [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+  margin: 0 0 5px 0;
+`
+
+const EventDescription = styled.p`
+  font-size: 18px;
```
Comment (FemmeQueenInSTEM) [inline src/pages/events.tsx]: And here too.

**Review by walsquared — CHANGES_REQUESTED** [review]
On top of the slight tweaks below, you'll need to add the "Events" button to the navigation in the footer.

**Review by walsquared — APPROVED** [review]
Much better. I can add the event tomorrow morning.

**Pull request merged**